code/modules/antagonists/eldritch_cult/knowledge/rust_lore.dm

```diff
@@ -119,7 +119,7 @@
 /datum/eldritch_knowledge/final/rust_final
 	name = "Rustbringer's Oath"
 	desc = "Bring 3 corpses onto the transmutation rune. After you finish the ritual rust will now automatically spread from the rune. Your healing on rust is also tripled, while you become more resillient overall."
```

Comment:
Aren't you raising the ratio here? the desc should reflect that

Reply:
I raised the base as well , so it is in fact exactly 3x when this is added.